util/rich-indexer/src/store.rs

```diff
@@ -75,15 +70,23 @@ impl SQLXPool {
                 let pool = pool_options.connect_with(connection_options).await?;
                 log::info!("PostgreSQL is connected.");
                 self.pool
-                    .set(pool)
+                    .set(pool.clone())
                     .map_err(|_| anyhow!("set pool failed"))?;
                 if require_init {
                     self.create_tables_for_postgres().await?;
                 }
                 self.db_driver = DBDriver::Postgres;
-                Ok(())
+                pool
             }
-        }
+        };
+
+        // Run migrations
+        log::info!("Running migrations...");
+        let migrator = Migrator::new(Path::new(SQL_MIGRATIONS)).await?;
+        migrator.run(&pool).await?;
```

Comment:
How long will this action take on the mainnet?

Reply:
About 40 minutes